packages/otel/src/sdk.ts

```diff
@@ -117,6 +117,9 @@ export class Sdk {
           process.env.VERCEL_BRANCH_URL ||
           process.env.NEXT_PUBLIC_VERCEL_BRANCH_URL ||
           undefined,
+        "vercel.deployment_id":
```

Comment:
This value is now set as the `version` attribute. Would you still like this additional attribute? If yes, could you pls run `changeset`?

Reply:
@dvoytenko: Yes, this is ideally a separate attribute for clarity for end-users, and to make automated processing of resource data more reliable.

I have now run `changeset`